### ISSUE TITLE
README mentions optional load of $HOME/.ossh.rb, but ossh requires it

### ISSUE THREAD

**Issue by deoren** [kt97679/one-ssh#1]
Hi,

I saw mention of your project, so I cloned the repo to give it a try. I installed bundler, ran `bundle install --path vendor/bundle` and then used `bundle` to execute the script. 

From the README: 

> On the start ossh tries to load $HOME/.ossh.rb file if it exists. 

When I try to run the script I see this:

`deoren@testbox:~/dev/one-ssh$ bundle exec bin/ossh`
```
bundler: failed to load command: bin/ossh (bin/ossh)
LoadError: cannot load such file -- /home/deoren/.ossh.rb
  bin/ossh:9:in `require'
  bin/ossh:9:in `<top (required)>'
```

`deoren@testbox:~/dev/one-ssh$ bundle list`
```
Gems included by the bundle:
  * bracecomp (0.1.2)
  * bundler (1.16.1)
  * em-ssh (0.8.1)
  * eventmachine (1.2.5)
  * highline (1.7.10)
  * net-ssh (4.2.0)
  * ossh (1.0.0)
```

The workaround was just to create the file:

`deoren@testbox:~/dev/one-ssh$ touch $HOME/.ossh.rb`
`deoren@testbox:~/dev/one-ssh$ bundle exec bin/ossh`
```
No command specified
No host file or host string specified
Please use -? for help
```

`deoren@testbox:~/dev/one-ssh$ bundle exec bin/ossh -?`
```
Usage: ossh [options]
    -p, --par PARALLELISM            How many hosts to run simultaneously (default: 256)
    -C, --command-file COMMAND_FILE  File with commands to run
    -c, --command COMMAND            Command to run
    -A, --askpass                    Prompt for a password for ssh connects (default: use key based authentication)
    -l, --user USER                  Username for connections (default: $LOGNAME)
    -t, --timeout TIMEOUT            Timeout for operation, 0 for no timeout (default: 0)
    -H, --host HOST_STRING           Add the given HOST_STRING to the list of hosts.
                                     HOST_STRING can contain multiple hosts separated by space, brace expansion can be used.
                                     E.g. "host{1,3..5}.com" would expand to "host1.com host3.com host4.com host5.com"
                                     This option can be used multiple times.
    -h, --hosts HOST_FILE            Read hosts from the given HOST_FILE.
                                     Each line in the HOST_FILE should be like HOST_STRING above.
                                     This option can be used multiple times.
    -o, --port PORT                  Port to connect to (default: 22)
    -n, --noresolve                  Don't resolve ip addresses to names
    -P, --preconnect                 Connect to all hosts before running command
    -i, --ignore-failures            Ignore connection failures in the preconnect mode (default: false)
    -k, --key PRIVATE_KEY            Use this private key.
                                     This option can be used multiple times
    -?, --help                       Show help
```

In short, the README says the customization file is optional, but the script seems to say otherwise.

**Comment by kt97679** [comment]
Hi @deoren , thanks for catching this and sorry for issues. This looks like a changed behavior in recent versions of ruby since it worked with older rubies. I just fixed this, please check if issue is resolved for you.

**Comment by deoren** [comment]
@kt97679 No worries, thanks for the quick response.

I pulled in the latest changes, removed the empty `$HOME/.ossh.rb` file I created earlier and ran the ossh script again. No errors or warnings this time and the script ran as expected.

I look forward to using this tool. Thank you for taking the time to release it for others to use.
